client/my-sites/plans/jetpack-plans/product-store/style.scss

```
@@ -20,3 +23,15 @@
         font-weight: 600;
     }
 }
+
+.jetpack-product-store__pricing-banner {
+	height: 120px;
+
+	margin-bottom: 1.25rem;
+
+	@include break-large {
+		height: 54px;
+
+		margin-bottom: 0;
+	}
+}
```

What about using this scss?
```suggestion
    
    &__pricing-banner {
        height: 120px;
    
        margin-bottom: 1.25rem;
    
        @include break-large {
            height: 54px;
    
            margin-bottom: 0;
        }
    }
}
```

---

I was hesitating to use nested CSS as I remembered it was not advised in the [CSS coding guidelines](https://wpcalypso.wordpress.com/devdocs/docs/coding-guidelines/css.md#nesting).